source/adapters/level_zero/context.cpp

```diff
@@ -174,7 +175,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urContextSetExtendedDeleter(
   std::ignore = Context;
   std::ignore = Deleter;
   std::ignore = UserData;
-  urPrint("[UR][L0] %s function not implemented!\n", __FUNCTION__);
+  logger::error("[UR][L0] {} function not implemented!", __FUNCTION__);
```

Comment:
Please, test these `__FUNCTION__` prints with different compilers, perhaps create a test case for such a print to verify if this works with compilers tested in UR CI.

Reply:
Why do you think that it will not work?